packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.10
```

Comment:
Where the change is only to `example` (so, app-facing plugins with federated implementations), the change is exempt from version updates, because it doesn't affect clients.